### PR TITLE
Application for goldFox

### PR DESCRIPTION
**Pull request by jianfengxiaojianfeng**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/crustio/Crust-Grants-Program/blob/main/applications/application_template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A Ethereum (USDT/CRU) address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md).
- [x] I accept the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md) and submit the PR.
- [x] The total funding amount of the project is below USDT $30k.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] The grant will only be announced once we've successfully [delivered](https://github.com/crustio/Crust-Grant-Milestone-Delivery) the first milestone.

### CODE REVIEW

File: applications/application_template.md [hunk 1]
```diff
@@ -1,128 +0,0 @@
...
-* **Project Name:** Name of your project 
-* **Team Name:** Legal name of your team
-* **Payment Address:** Ethereum address 
-* **Receivable Token:** CRU or USDT
-
-> ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
-
-> ⚠️ *When you submit an application for a Grant，You confirm that you have read these Terms and Conditions carefully and that you agree to be bound by them.*
-
-### Overview
```
Comment (yyd106) [inline applications/application_template.md:12]: You can't remove or change the original file, this is described in [Application Process](https://github.com/crustio/Crust-Grants-Program#1-application) clearly right?

File: applications/application_template.md [hunk 2]
```diff
@@ -1,128 +0,0 @@
-# Crust Grant Proposal
```
Comment (yyd106) [inline applications/application_template.md:1]: You can't remove or change the original file, this is described in [Application Process](https://github.com/crustio/Crust-Grants-Program#1-application) clearly right?

File: applications/application_GoldFox.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Web3、React、Solidity、Truffle、Go
+ 
+### Ecosystem Fit 
+We support the GPL open source protocol
+ 
+Help us locate your project in the Crust landscape and what problems it tries to solve by answering each of these questions:
+ 
+* Where and how does your project fit into the ecosystem? 
+* What need(s) does your project meet? 
+* Are there any other projects similar to yours in the I
```
Comment (yyd106) [inline applications/application_GoldFox.md:39]: Please complete your application before submitting your PR

Comment (yyd106) [reply applications/application_GoldFox.md:39]: You can't remove or change the original file, this is described in Application Process clearly right?

Comment (jianfengxiaojianfeng) [reply applications/application_GoldFox.md]: Ok

**Comment by yyd106** [comment]
Projects regarding token&finance are not included in Crust tech stack. I will close this PR.
If you want to contribute any idea on the improvement of Crust ecosystem applications or the Crust protocol, please feel free to submit.

**Pull request closed without merging**